src/xtopcom/xtxpool_v2/src/xtxpool.cpp

```diff
@@ -44,6 +45,18 @@ int32_t xtxpool_t::push_send_tx(const std::shared_ptr<xtx_entry> & tx) {
     if (table == nullptr) {
         return xtxpool_error_account_not_in_charge;
     }
+
+    if (nullptr == table->query_tx(tx->get_tx()->get_tx_hash())) {
+        auto f = [tx, table]() {
+            assert(tx != nullptr && table != nullptr);
+            return table->push_send_tx(tx) == xsuccess;
+        };
+        data::xmessage_t msg{common::xaccount_address_t{tx->get_tx()->get_account_addr()}, tx->get_tx(), f};
+        if (data::xpreprocess::instance().send(msg)) {

```

Comment:
add a comment to explain that normal transactions will push into txpool immediately